scripts/build-probe-binaries

```diff
@@ -607,8 +620,18 @@ if [ -z "$KERNEL_TYPE" ]; then
 	#fi
 
 	echo "Success."
+elif [ "CONTAINER-Fedora-Atomic" = "$EXTRA_BUILD_INSTRU" ]; then
```

Comment:
why is this block required?

Reply:
so is this change required now or only for the future? if required only for the future, please remove it so that this PR is limited to what is required now.

Reply:
we need it here now for testing the fedora atomic build. I prefer to keep it to make testing easy.